dietpi/dietpi-software

```diff
@@ -12318,6 +12324,40 @@ _EOF_
 			Download_Install "$(curl -sSfL 'https://api.github.com/repos/rclone/rclone/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/rclone-v[^\"\/]*linux-$arch.deb\"/{print \$4}")"
 
 		fi
+		
+		software_id=206 # Homer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+		
+		Banner_Installing
+		
+		# installing nginx and ensuring it starts at boot
+		sudo apt install nginx -y
```

Comment:
this is not needed as we have a direct install option for Nginx. It would need to be added as dependency.

Reply:
Also, we do generally not force a specific webserver. home should run well with Apache and Lighttpd as well.

Reply:
> And generally no sudo is required in this script, it can only run as root anyway

oh thats good then, i wasn't too sure when making the script so did it just in case. I'll remove the sudo's